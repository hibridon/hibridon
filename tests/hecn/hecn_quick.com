inp=Hecn_quick.inp
job=hecnq
noprin=t
show
run
printc
hypxsc,,1,2,0,2
exit
