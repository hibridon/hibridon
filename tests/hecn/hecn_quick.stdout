
 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V xdate

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
SYS=Debian bullseye/sid ( 5.4.0-80-generic); MACH=
CC=; FC=



 --------------------------------------------------------------------------
 Hibridon>  Hibridon>  Hibridon>  Hibridon>      *** Parameters (scattering):
 JTOT1  =   0        JTOT2  =   2        JTOTD  =   1        JLPAR  =   0
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   0
 FSTFAC =  1.500     RINCR  =  1.000     RCUT   =  80.00     RENDAI =  60.00    
 RENDLD =  25.00     RSTART =  3.500     SPAC   = 2.0000E-02 TOLAI  =  1.150    
 XMU    =   3.46870
 NOUT: 21; JOUT:  0   1   2   3   4   5   6   7   8   9  10  11  12  13  14  15  16  17  18  19
 20
 INDOUT:   -1    1
     *** 2-SIGMA system parameters:
 NTERM  =   1        NRMAX  =   8        NPAR   =   2        ISYM   =   1
 IGU    =   1        ISA    =   1
 BROT   =  1.891     GSR    = 7.2549E-03 DROT   = 6.4000E-06 HROT   =  0.000    
 LAMMIN:    1
 LAMMAX:   12
 MPROJ:     0
     *** Flags:
 AIRYFL= T    BASTST= F    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= T    NOPRIN= T    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= T    WRSMAT= T    WRXSEC= T    BOUNDC= F
 ** Maximum Channels:   81; Maximum Anisotropic Terms:  2000
 ** Energies:       40.000000
 ** Label:       HeCN(X2Sigma) RCCSD(T) Lique Spielfiede        
 ** Pot name:      He-CN(2Sigma) CCSDT PES                         
 ** Input File:  Hecn_quick.inp
 ** Output file: Outpt
 ** Jobname:     Hecnq
 Hibridon>  NOPRIN = .TRUE., SO IPRINT SET TO -1
 ** TOTAL NUMBER OF ANISTROPIC TERMS IN POTENTIAL = 12 BUT NLAM=  0  NLAM WILL BE REDEFINED
 label : HeCN CASSCF+RCCSD(T) SWCC rCN=2.214ao 818 geom. 13 angles AVQZ+CT99@mid  12par.:
 nterms :            1
 ntblk          13
 SURFACE LOADED FROM FILE hecn_fitmlv.dat                                                                 

  LABEL:HeCN CASSCF+RCCSD(T) SWCC rCN=2.214ao 818 geom. 13 angles AVQZ+CT99@mid   NUMBER OF POTENTIALS (NTERMS):            1   TOTAL NUMBER OF BLOCKS:                  13   TOTAL NUMBER OF POINTS:                818   TOTAL NUMBER OF COEFFICIENTS:            12
 nangle          13
 ** J =    0 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:16 2021
 ** J =    1 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:17 2021
 ** J =    2 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:17 2021
 ** J =    0 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:18 2021
 ** J =    1 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:18 2021
 ** J =    2 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:01:19 2021
 ===============================================================================
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:    45
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    NaN %
      CURRENT DATE:  Aug 12 14:01:19 2021
 ===============================================================================
 Hibridon> 
 INTEGRAL CROSS SECTIONS READ FROM FILE Hecnq1.ics                              
 WRITTEN:   Aug 12 14:01:19 2021
 LABEL:      HeCN(X2Sigma) RCCSD(T) Lique Spielfiede        
 POT NAME:  He-CN(2Sigma) CCSDT PES                         

 ** INTEGRAL CROSS SECTIONS; IEN= 1 **
    RMU=   3.4687  E=    40.00  JLPAR=-1  JTOT-1=  0.5  JTOT-2=   2.5  JTOT-D=  1
 ** CC CALCULATION, JLPAR=-1 **

 ** COLUMN HEADINGS ARE INITIAL STATES, ROW HEADINGS ARE FINAL STATES **

           J=   0.5       1.5       2.5       3.5       0.5       1.5       2.5
   J    I | I=   -1        -1        -1        -1         1         1         1
 
  0.5    1   9.043E-02  5.576E-01  5.648E-02  2.911E-01  6.819E+00  9.125E-02  5.689E-01
  0.5   -1   7.014E+00  1.287E-01  9.782E-01  2.133E-01  8.190E-02  3.832E-01  3.249E-02
  1.5    1   7.663E-01  7.961E-02  3.415E-01  8.503E-02  1.652E-01  6.870E+00  1.695E-01
  1.5   -1   2.036E-01  9.788E+00  3.050E-01  5.313E+00  7.992E-01  6.301E-02  1.795E-01
  2.5    1   7.709E-02  2.691E-01  3.686E-02  5.840E-01  1.222E+00  2.011E-01  7.390E+00
  2.5   -1   1.403E+00  2.765E-01  1.403E+01  1.068E+00  7.338E-02  2.451E-01  2.229E-02
  3.5    1   1.159E-01  1.895E-02  8.842E-02  2.717E-02  9.211E-02  1.134E+00  2.208E-01
  3.5   -1   5.183E-02  8.157E-01  1.809E-01  2.108E+01  6.404E-02  1.033E-02  5.981E-02
  4.5    1   2.259E-03  1.001E-02  2.084E-03  3.364E-02  5.638E-02  3.092E-02  4.959E-01

           J=   3.5       4.5
   J    I | I=    1         1
 
  0.5    1   5.325E-02  2.081E-01
  0.5   -1   6.069E-02  7.551E-03
  1.5    1   1.187E+00  2.066E-01
  1.5   -1   1.570E-02  5.296E-02
  2.5    1   2.742E-01  3.932E+00
  2.5   -1   6.641E-02  9.993E-03
  3.5    1   1.136E+01  6.753E-01
  3.5   -1   3.456E-03  2.732E-02
  4.5    1   1.058E-01  3.463E+00
 Hibridon> 
% HYPERFINE-RESOLVED CROSS SECTIONS
%      LABEL:      HeCN(X2Sigma) RCCSD(T) Lique Spielfiede        
%      POT NAME:  He-CN(2Sigma) CCSDT PES                         
%      S-MATRICES READ FROM FILE Hecnq1.smt                              
%      WRITTEN:   Aug 12 14:01:16 2021
%      JTOT2:              2
%      ETOT:          40.000 CM(-1)

% NUCLEAR SPIN =  1.0

 Computing partial wave J_tot =  0.5, jlpar=   1
 Computing partial wave J_tot =  0.5, jlpar=  -1
 Computing partial wave J_tot =  1.5, jlpar=   1
 Computing partial wave J_tot =  1.5, jlpar=  -1
 Computing partial wave J_tot =  2.5, jlpar=   1
 Computing partial wave J_tot =  2.5, jlpar=  -1
 Computing partial wave J_tot =  3.5, jlpar=   1
 Computing partial wave J_tot =  3.5, jlpar=  -1
%     E(CM-1)     JI     INI   FI      JF     INF   FF      CROSS SECTION (ANG^2)
      40.000     0.5     1   0.5      0.5     1   0.5          6.6513E+00
      40.000     0.5     1   0.5      0.5     1   1.5          1.6745E-01
      40.000     0.5     1   0.5      0.5    -1   0.5          9.0995E-03
      40.000     0.5     1   0.5      0.5    -1   1.5          7.2796E-02
      40.000     0.5     1   0.5      1.5     1   0.5          6.5810E-02
      40.000     0.5     1   0.5      1.5     1   1.5          8.5692E-02
      40.000     0.5     1   0.5      1.5     1   2.5          1.3718E-02
      40.000     0.5     1   0.5      1.5    -1   0.5          1.2246E-02
      40.000     0.5     1   0.5      1.5    -1   1.5          1.6963E-01
      40.000     0.5     1   0.5      1.5    -1   2.5          6.1731E-01
      40.000     0.5     1   0.5      2.5     1   1.5          5.9201E-01
      40.000     0.5     1   0.5      2.5     1   2.5          5.4476E-01
      40.000     0.5     1   0.5      2.5     1   3.5          8.5602E-02
      40.000     0.5     1   0.5      2.5    -1   1.5          2.2184E-03
      40.000     0.5     1   0.5      2.5    -1   2.5          1.8421E-02
      40.000     0.5     1   0.5      2.5    -1   3.5          5.2736E-02
      40.000     0.5     1   1.5      0.5     1   0.5          8.3724E-02
      40.000     0.5     1   1.5      0.5     1   1.5          6.7350E+00
      40.000     0.5     1   1.5      0.5    -1   0.5          3.6398E-02
      40.000     0.5     1   1.5      0.5    -1   1.5          4.5498E-02
      40.000     0.5     1   1.5      1.5     1   0.5          8.3999E-03
      40.000     0.5     1   1.5      1.5     1   1.5          3.9764E-02
      40.000     0.5     1   1.5      1.5     1   2.5          1.1706E-01
      40.000     0.5     1   1.5      1.5    -1   0.5          1.9367E-01
      40.000     0.5     1   1.5      1.5    -1   1.5          3.1478E-01
      40.000     0.5     1   1.5      1.5    -1   2.5          2.9074E-01
      40.000     0.5     1   1.5      2.5     1   1.5          1.1145E-01
      40.000     0.5     1   1.5      2.5     1   2.5          3.3881E-01
      40.000     0.5     1   1.5      2.5     1   3.5          7.7211E-01
      40.000     0.5     1   1.5      2.5    -1   1.5          2.3349E-02
      40.000     0.5     1   1.5      2.5    -1   2.5          2.7477E-02
      40.000     0.5     1   1.5      2.5    -1   3.5          2.2549E-02
      40.000     0.5    -1   0.5      0.5     1   0.5          1.0048E-02
      40.000     0.5    -1   0.5      0.5     1   1.5          8.0382E-02
      40.000     0.5    -1   0.5      0.5    -1   0.5          6.9076E+00
      40.000     0.5    -1   0.5      0.5    -1   1.5          1.0675E-01
      40.000     0.5    -1   0.5      1.5     1   0.5          1.1758E-02
      40.000     0.5    -1   0.5      1.5     1   1.5          1.6266E-01
      40.000     0.5    -1   0.5      1.5     1   2.5          5.9184E-01
      40.000     0.5    -1   0.5      1.5    -1   0.5          8.0148E-02
      40.000     0.5    -1   0.5      1.5    -1   1.5          1.0484E-01
      40.000     0.5    -1   0.5      1.5    -1   2.5          1.8621E-02
      40.000     0.5    -1   0.5      2.5     1   1.5          1.6192E-03
      40.000     0.5    -1   0.5      2.5     1   2.5          1.9049E-02
      40.000     0.5    -1   0.5      2.5     1   3.5          5.6423E-02
      40.000     0.5    -1   0.5      2.5    -1   1.5          6.7868E-01
      40.000     0.5    -1   0.5      2.5    -1   2.5          6.2497E-01
      40.000     0.5    -1   0.5      2.5    -1   3.5          9.9629E-02
      40.000     0.5    -1   1.5      0.5     1   0.5          4.0191E-02
      40.000     0.5    -1   1.5      0.5     1   1.5          5.0238E-02
      40.000     0.5    -1   1.5      0.5    -1   0.5          5.3373E-02
      40.000     0.5    -1   1.5      0.5    -1   1.5          6.9610E+00
      40.000     0.5    -1   1.5      1.5     1   0.5          1.8568E-01
      40.000     0.5    -1   1.5      1.5     1   1.5          3.0180E-01
      40.000     0.5    -1   1.5      1.5     1   2.5          2.7877E-01
      40.000     0.5    -1   1.5      1.5    -1   0.5          1.0828E-02
      40.000     0.5    -1   1.5      1.5    -1   1.5          4.9384E-02
      40.000     0.5    -1   1.5      1.5    -1   2.5          1.4340E-01
      40.000     0.5    -1   1.5      2.5     1   1.5          2.4887E-02
      40.000     0.5    -1   1.5      2.5     1   2.5          2.9021E-02
      40.000     0.5    -1   1.5      2.5     1   3.5          2.3183E-02
      40.000     0.5    -1   1.5      2.5    -1   1.5          1.2842E-01
      40.000     0.5    -1   1.5      2.5    -1   2.5          3.8915E-01
      40.000     0.5    -1   1.5      2.5    -1   3.5          8.8570E-01
      40.000     1.5     1   0.5      0.5     1   0.5          7.2690E-02
      40.000     1.5     1   0.5      0.5     1   1.5          1.8556E-02
      40.000     1.5     1   0.5      0.5    -1   0.5          1.1762E-02
      40.000     1.5     1   0.5      0.5    -1   1.5          3.7148E-01
      40.000     1.5     1   0.5      1.5     1   0.5          4.9978E+00
      40.000     1.5     1   0.5      1.5     1   1.5          1.4129E+00
      40.000     1.5     1   0.5      1.5     1   2.5          4.5912E-01
      40.000     1.5     1   0.5      1.5    -1   0.5          7.4553E-03
      40.000     1.5     1   0.5      1.5    -1   1.5          1.0982E-02
      40.000     1.5     1   0.5      1.5    -1   2.5          4.4577E-02
      40.000     1.5     1   0.5      2.5     1   1.5          1.4144E-01
      40.000     1.5     1   0.5      2.5     1   2.5          4.2248E-02
      40.000     1.5     1   0.5      2.5     1   3.5          1.7400E-02
      40.000     1.5     1   0.5      2.5    -1   1.5          6.9968E-02
      40.000     1.5     1   0.5      2.5    -1   2.5          8.9708E-02
      40.000     1.5     1   0.5      2.5    -1   3.5          8.5377E-02
      40.000     1.5     1   1.5      0.5     1   0.5          4.7325E-02
      40.000     1.5     1   1.5      0.5     1   1.5          4.3921E-02
      40.000     1.5     1   1.5      0.5    -1   0.5          8.1353E-02
      40.000     1.5     1   1.5      0.5    -1   1.5          3.0189E-01
      40.000     1.5     1   1.5      1.5     1   0.5          7.0645E-01
      40.000     1.5     1   1.5      1.5     1   1.5          5.1066E+00
      40.000     1.5     1   1.5      1.5     1   2.5          1.0568E+00
      40.000     1.5     1   1.5      1.5    -1   0.5          5.4912E-03
      40.000     1.5     1   1.5      1.5    -1   1.5          2.3064E-02
      40.000     1.5     1   1.5      1.5    -1   2.5          3.4460E-02
      40.000     1.5     1   1.5      2.5     1   1.5          4.6289E-02
      40.000     1.5     1   1.5      2.5     1   2.5          1.2071E-01
      40.000     1.5     1   1.5      2.5     1   3.5          3.4085E-02
      40.000     1.5     1   1.5      2.5    -1   1.5          6.1431E-02
      40.000     1.5     1   1.5      2.5    -1   2.5          8.3573E-02
      40.000     1.5     1   1.5      2.5    -1   3.5          1.0005E-01
      40.000     1.5     1   2.5      0.5     1   0.5          5.0505E-03
      40.000     1.5     1   2.5      0.5     1   1.5          8.6195E-02
      40.000     1.5     1   2.5      0.5    -1   0.5          1.9734E-01
      40.000     1.5     1   2.5      0.5    -1   1.5          1.8590E-01
      40.000     1.5     1   2.5      1.5     1   0.5          1.5304E-01
      40.000     1.5     1   2.5      1.5     1   1.5          7.0454E-01
      40.000     1.5     1   2.5      1.5     1   2.5          6.0123E+00
      40.000     1.5     1   2.5      1.5    -1   0.5          1.4859E-02
      40.000     1.5     1   2.5      1.5    -1   1.5          2.2973E-02
      40.000     1.5     1   2.5      1.5    -1   2.5          2.5183E-02
      40.000     1.5     1   2.5      2.5     1   1.5          1.1366E-02
      40.000     1.5     1   2.5      2.5     1   2.5          3.9500E-02
      40.000     1.5     1   2.5      2.5     1   3.5          1.5022E-01
      40.000     1.5     1   2.5      2.5    -1   1.5          4.4636E-02
      40.000     1.5     1   2.5      2.5    -1   2.5          7.7751E-02
      40.000     1.5     1   2.5      2.5    -1   3.5          1.2267E-01
      40.000     1.5    -1   0.5      0.5     1   0.5          1.7088E-02
      40.000     1.5    -1   0.5      0.5     1   1.5          5.4052E-01
      40.000     1.5    -1   0.5      0.5    -1   0.5          1.0129E-01
      40.000     1.5    -1   0.5      0.5    -1   1.5          2.7368E-02
      40.000     1.5    -1   0.5      1.5     1   0.5          9.4188E-03
      40.000     1.5    -1   0.5      1.5     1   1.5          1.3875E-02
      40.000     1.5    -1   0.5      1.5     1   2.5          5.6317E-02
      40.000     1.5    -1   0.5      1.5    -1   0.5          7.2139E+00
      40.000     1.5    -1   0.5      1.5    -1   1.5          1.6554E+00
      40.000     1.5    -1   0.5      1.5    -1   2.5          9.1889E-01
      40.000     1.5    -1   0.5      2.5     1   1.5          6.9805E-02
      40.000     1.5    -1   0.5      2.5     1   2.5          9.1137E-02
      40.000     1.5    -1   0.5      2.5     1   3.5          1.0816E-01
      40.000     1.5    -1   0.5      2.5    -1   1.5          2.0839E-01
      40.000     1.5    -1   0.5      2.5    -1   2.5          4.8850E-02
      40.000     1.5    -1   0.5      2.5    -1   3.5          1.9212E-02
      40.000     1.5    -1   1.5      0.5     1   0.5          1.1836E-01
      40.000     1.5    -1   1.5      0.5     1   1.5          4.3925E-01
      40.000     1.5    -1   1.5      0.5    -1   0.5          6.6245E-02
      40.000     1.5    -1   1.5      0.5    -1   1.5          6.2409E-02
      40.000     1.5    -1   1.5      1.5     1   0.5          6.9374E-03
      40.000     1.5    -1   1.5      1.5     1   1.5          2.9138E-02
      40.000     1.5    -1   1.5      1.5     1   2.5          4.3535E-02
      40.000     1.5    -1   1.5      1.5    -1   0.5          8.2769E-01
      40.000     1.5    -1   1.5      1.5    -1   1.5          7.4785E+00
      40.000     1.5    -1   1.5      1.5    -1   2.5          1.4820E+00
      40.000     1.5    -1   1.5      2.5     1   1.5          6.2205E-02
      40.000     1.5    -1   1.5      2.5     1   2.5          9.3014E-02
      40.000     1.5    -1   1.5      2.5     1   3.5          1.1389E-01
      40.000     1.5    -1   1.5      2.5    -1   1.5          6.0699E-02
      40.000     1.5    -1   1.5      2.5    -1   2.5          1.7687E-01
      40.000     1.5    -1   1.5      2.5    -1   3.5          3.8892E-02
      40.000     1.5    -1   2.5      0.5     1   0.5          2.8714E-01
      40.000     1.5    -1   2.5      0.5     1   1.5          2.7047E-01
      40.000     1.5    -1   2.5      0.5    -1   0.5          7.8439E-03
      40.000     1.5    -1   2.5      0.5    -1   1.5          1.2081E-01
      40.000     1.5    -1   2.5      1.5     1   0.5          1.8772E-02
      40.000     1.5    -1   2.5      1.5     1   1.5          2.9024E-02
      40.000     1.5    -1   2.5      1.5     1   2.5          3.1815E-02
      40.000     1.5    -1   2.5      1.5    -1   0.5          3.0630E-01
      40.000     1.5    -1   2.5      1.5    -1   1.5          9.8799E-01
      40.000     1.5    -1   2.5      1.5    -1   2.5          8.4939E+00
      40.000     1.5    -1   2.5      2.5     1   1.5          5.4863E-02
      40.000     1.5    -1   2.5      2.5     1   2.5          8.7015E-02
      40.000     1.5    -1   2.5      2.5     1   3.5          1.2723E-01
      40.000     1.5    -1   2.5      2.5    -1   1.5          1.2939E-02
      40.000     1.5    -1   2.5      2.5    -1   2.5          5.0111E-02
      40.000     1.5    -1   2.5      2.5    -1   3.5          2.1341E-01
      40.000     2.5     1   1.5      0.5     1   0.5          4.1332E-01
      40.000     2.5     1   1.5      0.5     1   1.5          1.5562E-01
      40.000     2.5     1   1.5      0.5    -1   0.5          1.0238E-03
      40.000     2.5     1   1.5      0.5    -1   1.5          3.1471E-02
      40.000     2.5     1   1.5      1.5     1   0.5          8.9401E-02
      40.000     2.5     1   1.5      1.5     1   1.5          5.8517E-02
      40.000     2.5     1   1.5      1.5     1   2.5          2.1554E-02
      40.000     2.5     1   1.5      1.5    -1   0.5          3.4925E-02
      40.000     2.5     1   1.5      1.5    -1   1.5          6.2245E-02
      40.000     2.5     1   1.5      1.5    -1   2.5          8.2347E-02
      40.000     2.5     1   1.5      2.5     1   1.5          5.6470E+00
      40.000     2.5     1   1.5      2.5     1   2.5          1.2399E+00
      40.000     2.5     1   1.5      2.5     1   3.5          5.0318E-01
      40.000     2.5     1   1.5      2.5    -1   1.5          7.5141E-03
      40.000     2.5     1   1.5      2.5    -1   2.5          8.4096E-03
      40.000     2.5     1   1.5      2.5    -1   3.5          6.3662E-03
      40.000     2.5     1   2.5      0.5     1   0.5          2.5355E-01
      40.000     2.5     1   2.5      0.5     1   1.5          3.1539E-01
      40.000     2.5     1   2.5      0.5    -1   0.5          8.0294E-03
      40.000     2.5     1   2.5      0.5    -1   1.5          2.4466E-02
      40.000     2.5     1   2.5      1.5     1   0.5          1.7803E-02
      40.000     2.5     1   2.5      1.5     1   1.5          1.0173E-01
      40.000     2.5     1   2.5      1.5     1   2.5          4.9935E-02
      40.000     2.5     1   2.5      1.5    -1   0.5          3.0398E-02
      40.000     2.5     1   2.5      1.5    -1   1.5          6.2048E-02
      40.000     2.5     1   2.5      1.5    -1   2.5          8.7070E-02
      40.000     2.5     1   2.5      2.5     1   1.5          8.2660E-01
      40.000     2.5     1   2.5      2.5     1   2.5          5.4735E+00
      40.000     2.5     1   2.5      2.5     1   3.5          1.0899E+00
      40.000     2.5     1   2.5      2.5    -1   1.5          5.6064E-03
      40.000     2.5     1   2.5      2.5    -1   2.5          7.8446E-03
      40.000     2.5     1   2.5      2.5    -1   3.5          8.8389E-03
      40.000     2.5     1   3.5      0.5     1   0.5          2.9882E-02
      40.000     2.5     1   3.5      0.5     1   1.5          5.3906E-01
      40.000     2.5     1   3.5      0.5    -1   0.5          1.7837E-02
      40.000     2.5     1   3.5      0.5    -1   1.5          1.4658E-02
      40.000     2.5     1   3.5      1.5     1   0.5          5.4993E-03
      40.000     2.5     1   3.5      1.5     1   1.5          2.1545E-02
      40.000     2.5     1   3.5      1.5     1   2.5          1.4243E-01
      40.000     2.5     1   3.5      1.5    -1   0.5          2.7058E-02
      40.000     2.5     1   3.5      1.5    -1   1.5          5.6979E-02
      40.000     2.5     1   3.5      1.5    -1   2.5          9.5480E-02
      40.000     2.5     1   3.5      2.5     1   1.5          2.5159E-01
      40.000     2.5     1   3.5      2.5     1   2.5          8.1744E-01
      40.000     2.5     1   3.5      2.5     1   3.5          6.3210E+00
      40.000     2.5     1   3.5      2.5    -1   1.5          3.1831E-03
      40.000     2.5     1   3.5      2.5    -1   2.5          6.6292E-03
      40.000     2.5     1   3.5      2.5    -1   3.5          1.2478E-02
      40.000     2.5    -1   1.5      0.5     1   0.5          2.5612E-03
      40.000     2.5    -1   1.5      0.5     1   1.5          5.3915E-02
      40.000     2.5    -1   1.5      0.5    -1   0.5          7.0960E-01
      40.000     2.5    -1   1.5      0.5    -1   1.5          2.6855E-01
      40.000     2.5    -1   1.5      1.5     1   0.5          7.3135E-02
      40.000     2.5    -1   1.5      1.5     1   1.5          1.2842E-01
      40.000     2.5    -1   1.5      1.5     1   2.5          1.3997E-01
      40.000     2.5    -1   1.5      1.5    -1   0.5          1.7242E-01
      40.000     2.5    -1   1.5      1.5    -1   1.5          1.0044E-01
      40.000     2.5    -1   1.5      1.5    -1   2.5          3.2114E-02
      40.000     2.5    -1   1.5      2.5     1   1.5          1.2426E-02
      40.000     2.5    -1   1.5      2.5     1   2.5          1.3907E-02
      40.000     2.5    -1   1.5      2.5     1   3.5          1.0528E-02
      40.000     2.5    -1   1.5      2.5    -1   1.5          1.0754E+01
      40.000     2.5    -1   1.5      2.5    -1   2.5          2.5511E+00
      40.000     2.5    -1   1.5      2.5    -1   3.5          7.2231E-01
      40.000     2.5    -1   2.5      0.5     1   0.5          1.4178E-02
      40.000     2.5    -1   2.5      0.5     1   1.5          4.2298E-02
      40.000     2.5    -1   2.5      0.5    -1   0.5          4.3564E-01
      40.000     2.5    -1   2.5      0.5    -1   1.5          5.4251E-01
      40.000     2.5    -1   2.5      1.5     1   0.5          6.2512E-02
      40.000     2.5    -1   2.5      1.5     1   1.5          1.1647E-01
      40.000     2.5    -1   2.5      1.5     1   2.5          1.6254E-01
      40.000     2.5    -1   2.5      1.5    -1   0.5          2.6944E-02
      40.000     2.5    -1   2.5      1.5    -1   1.5          1.9511E-01
      40.000     2.5    -1   2.5      1.5    -1   2.5          8.2919E-02
      40.000     2.5    -1   2.5      2.5     1   1.5          9.2711E-03
      40.000     2.5    -1   2.5      2.5     1   2.5          1.2972E-02
      40.000     2.5    -1   2.5      2.5     1   3.5          1.4617E-02
      40.000     2.5    -1   2.5      2.5    -1   1.5          1.7008E+00
      40.000     2.5    -1   2.5      2.5    -1   2.5          1.0237E+01
      40.000     2.5    -1   2.5      2.5    -1   3.5          2.0893E+00
      40.000     2.5    -1   3.5      0.5     1   0.5          3.0443E-02
      40.000     2.5    -1   3.5      0.5     1   1.5          2.6033E-02
      40.000     2.5    -1   3.5      0.5    -1   0.5          5.2085E-02
      40.000     2.5    -1   3.5      0.5    -1   1.5          9.2607E-01
      40.000     2.5    -1   3.5      1.5     1   0.5          4.4621E-02
      40.000     2.5    -1   3.5      1.5     1   1.5          1.0458E-01
      40.000     2.5    -1   3.5      1.5     1   2.5          1.9233E-01
      40.000     2.5    -1   3.5      1.5    -1   0.5          7.9475E-03
      40.000     2.5    -1   3.5      1.5    -1   1.5          3.2178E-02
      40.000     2.5    -1   3.5      1.5    -1   2.5          2.6485E-01
      40.000     2.5    -1   3.5      2.5     1   1.5          5.2638E-03
      40.000     2.5    -1   3.5      2.5     1   2.5          1.0963E-02
      40.000     2.5    -1   3.5      2.5     1   3.5          2.0634E-02
      40.000     2.5    -1   3.5      2.5    -1   1.5          3.6116E-01
      40.000     2.5    -1   3.5      2.5    -1   2.5          1.5669E+00
      40.000     2.5    -1   3.5      2.5    -1   3.5          1.2099E+01
 Hibridon> 